tests/gnmi/helper.py

```diff
@@ -195,6 +195,7 @@ def gnmi_set(duthost, ptfhost, delete_list, update_list, replace_list, cert=None
     cmd += '--xpath ' + xpath
     cmd += ' '
     cmd += '--value ' + xvalue
+    ptfhost.shell(f"ping {ip} -c 3", module_ignore_errors=True)
```

Comment:
How can ping improve the test stability?

Reply:
There is a chance that the network connection lost between PTF and switch due to table timeout, which lead to execution failure of py_gnmicli.py. The ping action would trigger arp and mac table refresh.